### ISSUE TITLE
Music style/feel for Project Porcupine

### ISSUE THREAD

**Issue by longtomjr** [TeamPorcupine/ProjectPorcupine#631]
This is a discussion thread to try and figure out what music feel and style we want to go for. I just want the PPMusicians to know what style and feel to have in mind when composing and playing.

Here is my first improv track. I just wanted to put something on tape. This is just to see if this style resonates with the team. https://soundcloud.com/longtomjr/space-in-time
It was inspired by the Bastion OST. 

Don't be afraid of anyone's feelings, at this time we are just trying to get a feel for what we want to go for.

**Comment by Tranberry** [comment]
If I had a valid vote I'd vote for 2001: A Space Odyssey 1968 - Movie Soundtrack like. But that might be a bit too bombastic and odd for a game.

**Comment by longtomjr** [comment]
because we are not story driven I am looking for something a bit more ambient. I will give the OST a listen and see what I think. :) Thanks for sharing!

**Comment by longtomjr** [comment]
Oh and another problem is that we do not have a symphonic orchestra at our disposal unfortunately. So I am also trying to keep it simple.

**Comment by Tranberry** [comment]
yeah, but as with art you can boil it down to a few notes I'm sure ;)

**Comment by ajahraus** [comment]
That's a good tune, though perhaps a bit somber for ambient music. The acoustic guitar brings with it a bit of a 'space western' vibe, which could be the aesthetic we end up with, but I'm not sure we're there yet. 

Now that I'm thinking about it, perhaps there may be a way to tie the ambient music to the recent in-game events or moods of the characters? If there was just an attack and a few people died, maybe a nice somber tune is just what's called for.

**Comment by ajahraus** [comment]
Would it be appropriate for us to link to songs that have the style/feeling we'd like to see emulated, even if we aren't the artist? Or is musical style too personal to crowd source like that? Maybe we need some executive orders to get us started in an appropriate direction.

**Comment by SquareMan** [comment]
I dig it! I'm not sure if it would fit as ambient, anytime music, but as @ajahraus said, i'm sure it could be tied into the eventsystem and trigger when appropriate.

**Comment by koosemose** [comment]
I'm not sure how feasible it would be, but it would interesting (somewhat of an addition to what @ajahraus suggested) to have some kind of rating of how the base was doing overall, that the music could tie in to. So aside from some kind of somber/mournful music when multiple people have died from an event, if the base was suddenly doing very well (steep increase in rating) a happier more upbeat music would be used, a sudden downswing would cause sad music, and if it was fairly rapidly going up and down (or had just had multiple "dangerous" events) it would be a more exciting music, in addition to a standard ambient track.

**Comment by longtomjr** [comment]
Thanks a lot for the replies! It really helps a ton!
We have already talked about a situation based Soundtrack. I am for this. Also feel free to link stuff you want to reference, it will help me understand what you mean a lot more :) 

Another question I have: Will we make the music all one style to get a certain feel, or will we mix and match (like soundsense for df). I don't really know in what camp I am, but for a FOSS project the latter might work better.

**Comment by satch3l** [comment]
Here is a track I'm working on. https://soundcloud.com/user-86473338/luna-wip 
It's more electronic style. But I'm gonna try make some more ambient ones as well.

**Comment by longtomjr** [comment]
I like it. I dunno if it will fit in a base-building game while playing though. This is just my two cents. I do like the track though :).
Maybe it is a good fit for the title screen if it is loopable. It can also be used with the events system, I just don't think it will work for background stuff.

I think the style might work, but it is a bit to jumpy :). I really like the instruments you used. You can try to add some more base end as well.

Thanks a LOT for sharing. Great work man. Happy to see someone working on stuff :)!

**Comment by misspellted** [comment]
"Space In Time"... I got kinda a "hmm, this item makes music... what can I do after a few weeks of plucking at it while I'm traveling in space to the space station" vibe.

Maybe the traveler is the only one, or of but a few on a ship heading out, and needs a little distraction from looking at all the same dots in the sky?

I liked it, although it felt, at times, obstructed. Like it didn't always flow from one section (measure?) to the next.. and sometimes the 'moving of the fingers' to setup the next chord didn't feel ... appropriate, or maybe too invasive/loud?

**Comment by lincore81** [comment]
How about something like Journey of the Sorcerer? 
https://www.youtube.com/watch?v=7rOMGIbY-9s

**Comment by Mizipzor** [comment]
I always assumed it would be something electronic. Fat, lofty pads. Drawn out chords, slight vibrato. Ambient. You know, space music! :)

**Comment by longtomjr** [comment]
thanks for all the feedback! There will be some original stuff! @misspellted do you lik the style, the space in time was just a sample track, nothing properly composed. do you think the feel and style will work for some tracks.

@Mizipzor How do you feel about the style and feel of the music, should it all be same style / feel or do you think we can make a mix of different styles/feels as long as it is spacey?

@lincore81 Thanks for the feedback. I like the song. Will use it a bit as inspiration and see what i can do. :)

**Comment by Mizipzor** [comment]
I would assume that we can mix music styles. But Im very picky though, Im one of those that usually turns off the music and pick my own track. :)

**Comment by quill18** [comment]
Not saying this is the style I'm looking for, but here's another possible source of inspiration:

"The Music of Battlestar Galactica for Solo Piano"
https://play.spotify.com/album/57FrnGmTJoXxqqInO2LIGQ

**Comment by longtomjr** [comment]
@quill18 Is it this https://www.youtube.com/watch?v=ir8U3mktf24 For those that live out of the reach of spotify. (From South Africa)

**Comment by satch3l** [comment]
If I could chose I would go for something like this https://www.youtube.com/watch?v=QAB59q43pbg&index=5&list=PLA1C13BE94FC21B88

(My favorite game orchestral music of all time! ;) )

Even though it might a bit much adrenalin pumping music, I'll probably look at this for inspiration. May have to tone it down a bit to fit a more base building game rather than a RTS combat game. ^^

**Comment by Mizipzor** [comment]
For me, the [music of Eve Online](https://www.youtube.com/watch?v=lEEmORyxpho) is the ultimate space music. Especially that opening segment of the video.

**Comment by Geoffrotism** [comment]
@satch3l  that track you made is F@#$@$ gold. My vote is putting it as main menu. get the user pumped right away.

**Comment by sboigelot** [comment]
@satch3l I agree with @Geoffrotism, very good work. A bit too fast pace for the menu thougth, it's more like a combat situation I think.

@Mizipzor :heart: Eve Online music

but.. yhea :heart: :heart: :heart: :heart: Totala music :D Orchestral music with modern toutch is always great, makes me think of https://www.youtube.com/watch?v=I1BJzKy8TbE
